### ISSUE TITLE
Restyle the Target Point Calculation Page

### ISSUE THREAD

**Issue by KPHowley** [hackforla/tdm-calculator#209]
### Overview
Restyle the Target Point Calculation Page to reflect the new design in Figma.

Follow up to #142

### Action Items
- [x] Re-style page 
- [x] Review with design team 
- [x] implement new design updates

### Resources/Instructions
<img width="1440" alt="A7 Target point Calculation@2x" src="https://user-images.githubusercontent.com/49739276/73114974-79763400-3ed4-11ea-8f61-07026d5fe0bc.png">

https://www.figma.com/file/qNgPc5lttpf0SWSwkbbWyT/TDM-Calculator?node-id=0%3A1

**Comment by ExperimentsInHonesty** [comment]
@mmorgs please provide update on
1. Progress
2. Blocks
3. Availability
4. ETA

**Comment by ExperimentsInHonesty** [comment]
@mmorgs please provide update on
1. Progress
2. Blocks
3. Availability
4. ETA

**Comment by mmorgs** [comment]
note to self: add class to 'page 4' and target child input in JSS

**Comment by ExperimentsInHonesty** [comment]
@bfojas separated the components for each step of the wizard.  That should help.

@mmorgs please provide update on
1. Progress
2. Blocks
3. Availability
4. ETA

P.s. we missed you last week!!!

**Comment by fyliu** [comment]
<img width="832" alt="Screen Shot 2020-03-02 at 11 28 22 AM" src="https://user-images.githubusercontent.com/1160105/75710973-6d6f5600-5c7a-11ea-97d1-2defc5cb0479.png">
@KPHowley please review the UI in staging. Here's a screenshot.

**Comment by KPHowley** [comment]
@mmorgs 
There's a few adjustments I'd love to make if possible.
- can we get rid of the yellow highlighting feature on this page, as it's not necessary 
- can we change the copy and wording to reflect what's in Figma
- can we change the input box to look like it does in this new mockup

**Figma File:** https://www.figma.com/file/VCVLtBFyN6mRnlk0jQfQDU/A7-Target-point-calculation?node-id=0%3A1

**Screen Shot:**
![A7 Target point Calculation](https://user-images.githubusercontent.com/49739276/76120887-492bb600-5fa7-11ea-950e-878f60d575a3.png)

**Comment by ExperimentsInHonesty** [comment]
Hi @mmorgs Can you give us an update on this.  Hope you are ok!
1. Progress
2. Blocks
3. Availability
4. ETA

**Comment by ExperimentsInHonesty** [comment]
@dazainus will create two prototypes and paste them here in the issue

**Comment by dazainus** [comment]
I've come up with 3 options for Target Point Calculation Page. Each option is shown with a screenshot image as well as a short description.

### Goal
- [ ] Make the 'Project Level' information more noticeable.


### Screenshots
#### OPTION 1
![OPTION 1  A7 Target Point Calculation](https://user-images.githubusercontent.com/55513459/77857989-a802da80-71b5-11ea-89d9-5c8e1fb532bc.png)
-  'Project Level' information is located on the left panel. 

#### OPTION 2
![OPTION 2  A7 Target Point Calculation](https://user-images.githubusercontent.com/55513459/77857994-ad602500-71b5-11ea-8947-0be8ed8de44a.png)
- 'Project Level' stays in the main content area, but with different styling in order to draw more attention.

#### OPTION 3
![OPTION 3  A7 Target Point Calculation](https://user-images.githubusercontent.com/55513459/77858001-b2bd6f80-71b5-11ea-8ea0-ff669eabe01a.png)
- Combination of Option 1 & 2. 'Project Level' is shown prominently on the left panel & the chart styling highlights the important pieces of information in the main context area.
- If this option is chosen, the written instruction(copywriting) needs to be revisited.


### Resources
- Figma file: https://www.figma.com/file/qNgPc5lttpf0SWSwkbbWyT/TDM-Calculator?node-id=1009%3A1779

**Comment by dazainus** [comment]
As a reference, sharing the screenshot of the original Target Point Calculation Page here:

#### ORIGINAL
![ORG  A7 Target Point Calculation](https://user-images.githubusercontent.com/55513459/77858338-bce06d80-71b7-11ea-95aa-9bb34399b409.png)

**Comment by KPHowley** [comment]
@Diane,  I like the options and just have a few comments before we can share with LADOT:

- First can we change the naming to reflect what is in this new issue #331 
- Also for the top portion of the sidebar, can we have it reflect the two values - "earned points" and "target points" which are on the site now (I know my mock up didn't reflect that! but it was a bit behind where we were in development)

**Comment by dazainus** [comment]
@KPHowley, I will start working on this. Do you need these to be completed by certain date?

**Comment by dazainus** [comment]
@KPHowley, I've updated the designs to reflect the requested changes. It seems like I just picked up on previous Figma file which was not entirely up-to-date. Please review and let me know if there is anything to be revised.

### OBJECTIVES

- [ ] Update naming in the main content area(as addressed in issue #331)
- [ ] Update naming on the sidebar (to "earned points" & "target points")

### DESIGN MOCKUPS:
#### OPTION 1 (UPDATED)
![OPTION 1  A7 Target Point Calculation](https://user-images.githubusercontent.com/55513459/78422411-eb28d780-7613-11ea-8ed9-2e004e8c1baa.png)

- Sidebar shows both TDM Calculation & Project Level

#### OPTION 2 (UPDATED)
![OPTION 2  A7 Target Point Calculation](https://user-images.githubusercontent.com/55513459/78422416-f419a900-7613-11ea-9631-9adc808ca153.png)
- Sidebar shows only TDM Calculation
- Project level is addressed in the content area

#### OPTION 3 (UPDATED)
![OPTION 3  A7 Target Point Calculation](https://user-images.githubusercontent.com/55513459/78422649-a4d47800-7615-11ea-8b07-935c70817ca7.png)

- Sidebar shows both TDM Calculation & Project Level
- TDM calculation & Project Level areas have their own dashboard titles 
  -  _**=> Will this provide clarification or confusion?**_


### RESOURCE
- Figma file: https://www.figma.com/file/qNgPc5lttpf0SWSwkbbWyT/TDM-Calculator?node-id=1009%3A1779

**Comment by ExperimentsInHonesty** [comment]
@Dsomers74 and @ramccarr - Please check out the designs [here](https://github.com/hackforla/tdm-calculator/issues/209#issuecomment-608996926), and let us know what feedback you might have about these design options.  These designs are in response to a request for a page that is clearer in its communication about project level.    You can write your questions/comments directly in the issue.

**Comment by ramccarr** [comment]
I prefer option 2.

**Comment by dazainus** [comment]
@dazainus - create Figma prototypes to visualize & test the process.

**Comment by Dsomers74** [comment]
I also prefer Level 2. Looks great!

**Comment by ExperimentsInHonesty** [comment]
Issue #337 will be the issue for implementing changes

**Comment by KPHowley** [comment]
@dazainus From meeting 4/15 - David Somers asked for Target points to be displayed in the blue bar, can we get an updated mockup. 

When approved, add this image to #337 , and notify Louis for implementation.

**Comment by dazainus** [comment]
Reflecting the previous comments and feedbacks, there are updated design.
@KPHowley - Please let me know if this looks good!

![OPTION6-TDM-target-points](https://user-images.githubusercontent.com/55513459/80052873-88b74e80-84d0-11ea-9e2b-11f226163f5f.jpg)

**Comment by KPHowley** [comment]
Per David and Rosemary, we should implement this change

**Comment by ExperimentsInHonesty** [comment]
This has been approved by stakeholder and implemented on staging.  Waiting final signoff
![Screen Shot 2020-05-02 at 10 46 54 AM](https://user-images.githubusercontent.com/37763229/80872912-d3239280-8c69-11ea-818d-12b9c556c20b.png)
